packages/app/source/server/projections/todos-projection.js

```diff
@@ -1,7 +1,7 @@
 Space.eventSourcing.Projection.extend('Todos.TodosProjection', {
 
-  collections: {
-    todos: 'Todos.TodoLists'
+  dependencies: {
```

Comment:
@qejk If you move this into the `collections` object I believe any repositories backed by `Mongo.Collection`s will work without any changes as `ProjectionRebuilder` [uses this API](https://github.com/meteor-space/event-sourcing/blob/master/source/server/infrastructure/projection-rebuilder.coffee#L54). We could rename it but `collections` is generic enough in my mind, however we could make a backwards compatible change and depreciate if not.

To support other persistence technologies we will need to flesh out the projection repository api to support the rebuilding calls in [rebuild](https://github.com/meteor-space/event-sourcing/blob/master/source/server/infrastructure/projection-rebuilder.coffee#L11-L47)